src/app.py

```diff
@@ -130,20 +157,11 @@ async def get(tick_number: int) -> Dict[str, Union[int, str, List[List[str]]]]:
 
 
 @app.post("/add_object")
-async def add(r: AddObjectRequest):
+async def add(r: AddObjectRequest, background_tasks: BackgroundTasks):
     object = Object.deserialize(r.serialized_object)
-    state.scene.add_entity(object)
+    state.scene.add_to_pending_actions(CreateObject(r.serialized_object).serialize())
     if object.tick != "":
```

Comment:
Would be cool to have a comment here explaining that we only spawn processes for agents with code
Also, we really need to change the name of this attribute (`tick`, I mean) in the future xD